src/main/java/core/basesyntax/service/operation/SupplyOperationHandler.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+
+public class SupplyOperationHandler implements OperationHandler {
+    @Override
+    public void calculate(Fruit fruit, FruitTransaction transaction) {
+        fruit.setQuantity(fruit.getQuantity() + transaction.getQuantity());
+    }
```

Comment:
```suggestion
    private final FruitDao fruitDao;

    public SupplyOperationHandler(FruitDao fruitDao) {
        this.fruitDao = fruitDao;
    }

    @Override
    public void process(FruitTransaction transaction) {
        Fruit fruit = fruitDao.get(transaction.getFruitName());
        int newQuantity = fruit.getQuantity() + transaction.getQuantity();
        fruit.setQuantity(newQuantity);
        fruitDao.update(fruit);
    }
```

I meant smth like this (same for other handlers)